shared/fonts/font_pack.cpp

```diff
@@ -0,0 +1,56 @@
+#include "fonts.h"
+
+using enigma::rect_packer::pvrect;
+using enigma::rect_packer::pack_rectangles;
+
+namespace enigma {
+namespace fonts {
+unsigned char* font_pack(RawFont& font, unsigned& textureW, unsigned& textureH) {
+ 
+  if (!pack_rectangles(font.glyphs, textureW, textureH))
+    return nullptr;
+
+  // intialize blank image (the caller is responisible for deleting pxdata)
+  unsigned char* pxdata = new unsigned char[textureW * textureH * font.channels];
+  for (unsigned y = 0; y < textureH; ++y) {
+    for (unsigned x = 0; x < textureW; ++x) {
+      unsigned index = y * textureW + x;
+      for (unsigned i = 0; i < font.channels; ++i)
+        pxdata[index+1] = 0;
```

Comment:
As discussed on Discord, this is a typo, you meant to add the control variable i not the number one.